### ISSUE TITLE
Basic Authentication: AuthName directive

### ISSUE THREAD

**Issue by Jay2k1** [Fyrd/caniuse#2370]
It's not exactly HTML or CSS, but I noticed current Chrome doesn't show the AuthName text from basic authentication used for example in a .htaccess file.

**Comment by Jay2k1** [comment]
Agreed. At this point it's not only Chrome but Chromium it seems, making Opera, Vivaldi and Safari also affected. There is an [issue on the Chromium bugtracker](https://bugs.chromium.org/p/chromium/issues/detail?id=544244&q=-has%3Afleet&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Cr%20Status%20Owner%20Summary%20OS%20Modified) about this. This means that of the major browsers, only Firefox, IE and Edge show the AuthName.

**Comment by francwalter** [comment]
Unfortunately, Firefox 93 has stopped to show AuthName too 😢 
It was such a nice feature, as I used to show password hints in this AuthName field 😏  - not the best was to secure things, though.
